test/unit/file-require-transform.test.js

```diff
@@ -355,6 +355,16 @@ suite('FileRequireTransform', () => {
     )
   })
 
+  test('Modern JS syntax features', () => {
```

Comment:
Can we make this test name more specific? "Modern" is a moving target.

```suggestion
  test('Object spread properties', () => {
```

Reply:
good point 😅